doc/build/spelling_wordlist.txt

```diff
@@ -0,0 +1,220 @@
+Aiosqlite
+CTE
+DBAPI
+DDL
+DML
+DSN
+FK
+Firebird
+Firebolt
+Hana
+Informix
+InstrumentedList
+JQuery
+Jython
+Netezza
+ODBC
+ORM
+Pysqlite
+Solr
+Sybase
+Teradata
+Vinay Sajip
+accessor
+ad-hoc
+anonymizing
+args
+async
+atomicity
+attrs
+autobegin
+autobegins
+autobegun
+autoclose
+autocommit
+autoexpire
+autoflush
+autoflushing
+autogenerate
+autoincrement
+autoincrementing
+autoload
+automations
+awaitable
+backend
+backends
+backref
+backreferences
+backrefs
+boolean
+cPython
+cacheable
+callables
+centric
+changelog
+changelogs
+changeset
+charset
+checkfirst
+checkins
+classmethod
+classmethods
+classwide
+codepath
+composable
+configurability
+configurational
+cutocommit
+datatype
+datatypes
+datatyping
+datetime
+deassociate
+deassociated
+deduplicate
+deduplicates
+deduplication
+deprecations
+dereference
+dereferenced
+deserialization
+deserialized
+deserializer
+deserializers
+deterministically
+dicts
+entrypoint
+eventlet
+executemany
+executional
+freeform
+fu
+generatively
+gevent
+greenlets
+hardcoded
+hashable
+hexstring
+hoc
+hostname
+hstore
+html
+incrementing
+inspectable
+instrumentedlist
+invalidations
+ish
+isouter
+iterable
+kwargs
+lexically
+locatable
+macOS
+mariadbconnector
+md5
+memoize
+metaclass
+metaprogramming
+mixin
+mixins
+monkeypatch
+monkeypatched
+monkeypatches
+monkeypatching
+multithreaded
+multivalued
+mutexes
+mysql-python
+mysqldb
+namespace
+namespaces
+natively
+nullability
+nullable
+objectstore
+oid
+ordinality
+parenthesization
+performant
+picklable
+polymorphically
+positionally
+postgresql
+pre
+pre-ping
+prepended
+prerelease
+programmatically
+proxied
+proxying
+psycopg
+py
+pymysql
+pyodbc
+pysqlcipher
+reconnection
+reconstructor
+reusability
+rowcount
+runtime
+savepoint
+savepoint
+savepoints
+schemas
+selectable
+selectables
+serializable
+serialization
+serialized
+serializer
+serializers
+setuptools
+sharding
+sql
+sqlalchemy
+sqlite
+strategized
+stringification
+stringified
+stringify
+stringifying
+subclass
+subclasse
+subclassed
+subclasses
+subclassing
+subcomponent
+subcomponents
+sublicense
+subqueries
+subquery
+subtransactions
+subtype
+superclass
+superclasses
+syntaxes
+tablename
+tablespace
+tablespaces
+threadlocal
+threadsafe
+timezones
+transactional
+transactional
+unary
+unbuffered
+uncacheable
+uncompiled
+undeferred
+unformatted
+unhandled
+unhashable
+unicode
+uniquing
+unmarshalling
+userland
+validator
+visitable
+walkthrough
+zxJDBC
+zxJDBC
```

Comment:
this seems duplicate

Reply:
Good catch - I'll dedupe before marking the PR as non-draft